lively.collab/comments/commentMorph.js

```diff
@@ -217,11 +217,38 @@ export class CommentMorph extends Morph {
 
     this.isInEditMode = false;
 
+    this.resetButtons();
+    this.ui.editSaveButton.tooltip = 'Edit Comment';
+
     this.ui.commentTextField.readOnly = true;
     this.ui.commentTextField.fill = Color.rgb(240, 243, 244);
     this.ui.commentTextField.borderStyle = 'none';
   }
 
+  resetButtons () {
```

Comment:
Maybe `enableButtons` would be clearer?

Reply:
We discussed this briefly yesterday, but I have to confess that our reasoning slipped my mind. :/ Maybe @frcroth remembers?

Also, thinking about this again I am not super happy with either of these options. What do you think of e.g. `enabledButtonAppearance` and `disabledButtonAppearance`? Since we only make visual changes I think that should be reflected in the name.